dts/apq8016-samsung-r02.dts

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8916.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <247 0>;
+	qcom,board-id = <0xF708FF01 2>;
+
+	// Unfortunately, Samsung uses the same qcom,board-id for all its
+	// APQ8016 devices. Therefore, it will always pick this shared device
+	// tree and we need to do more manual work to differentiate the devices.
+  matissewifive-eur {
```

Comment:
I think there is only an -eur version, so we might as well just call it `matissevewifi`